ui/pages/send/send-content/send-asset-row/send-asset-row.component.js

```diff
@@ -31,13 +34,41 @@ export default class SendAssetRow extends Component {
 
   state = {
     isShowingDropdown: false,
+    sendableTokens: [],
   };
 
+  async componentDidMount() {
+    const sendableTokens = this.props.tokens.filter((token) => !token.isERC721);
+    this.setState({ sendableTokens });
+  }
+
   openDropdown = () => this.setState({ isShowingDropdown: true });
 
   closeDropdown = () => this.setState({ isShowingDropdown: false });
 
-  selectToken = (token) => {
+  clearUnsendableAssetError = () => {
+    this.props.setUnsendableAssetError(false);
+    this.props.updateSendErrors({
+      unsendableAssetError: null,
+      gasLoadingError: null,
+    });
+  };
+
+  selectToken = async (token) => {
+    if (token && token.isERC721 === undefined) {
+      const updatedToken = await this.props.updateTokenType(token.address);
```

Comment:
I'm hoping for some feedback from others on this, but should we be doing this type of property/token update in a send component?  I feel like we may want to do this at a higher level, but I'd really appreciate feedback from others.

Reply:
So per this [thread](https://consensys.slack.com/archives/G1L7H42BT/p1622590868094100) (albeit with little feedback) this was the simplest solution to allow for multiple attempts to update the tokenType in the case of network issues. Its not perfect but I'm not sure if there is a better place for it. Open to suggestions though!

Reply:
@darkwing ^